.gitignore

```diff
@@ -104,6 +104,9 @@ KLU_module_deps
 # setup
 setup.log
 
+# test
+test.c
```

Comment:
why was this added?

Reply:
This file is generated every time someone runs the `run-tests.py`, so I thought to put it in the gitignore as it is only used locally and should not be pushed. Should I undo this?

Reply:
ok, no problem that's fine